src/workerd/api/tests/js-rpc-test.js

```diff
@@ -302,6 +302,38 @@ export class MyService extends WorkerEntrypoint {
   async roundTrip(value) {
     return value;
   }
+
+  async returnHeaders() {
+    let result = new Headers();
+    result.append("foo", "bar");
+    result.append("Set-Cookie", "abc");
+    result.append("set-cookie", "def");
+    result.append("corge", "!@#");
+    result.append("Content-Length", "123");
```

Comment:
Might be a bit pedantic but a test that serializes an empty `Headers()` might be worthwhile.

Reply:
f64d6252